test/test_jit.py

```diff
@@ -6572,6 +6572,7 @@ def forward(self):
         # Specialized error for Tensors
         class S(torch.jit.ScriptModule):
             def __init__(self):
+                super(S, self).__init__()
```

Comment:
Nit: this seems to be an unrelated change.

Reply:
It is related. Separating the C/Python type made this test fail because it forgot to call super. I fixed the tests where this was the case and our library is much more verbose about saying that you forgot to call __init__ now.